docs/source/guide.rst

```diff
@@ -27,7 +27,7 @@ Get all argument list:
                             exporting default config to file (default: config.json)
       --export_log_config [PATH]
                             exporting default logger config to file (default: log.yaml)
-      --rules [PATH]        path of rule config file (default: credsweeper/rules/config.yaml)
+      --rules [PATH]        path of rule config file (default: credsweeper/rules/config.yaml). severity:['critical', 'high', 'medium', 'low', 'info'] type:['keyword', 'pattern', 'pem_key']
```

Comment:
Why are these descriptions merged together?
--rules are about the file path, not file content details.

Reply:
There was a question about severity... Possibly it might be removed in future